presto-product-tests/src/main/resources/sql-tests/testcases/horology_functions/atTimeZoneAccessible.sql

```diff
@@ -1,3 +1,3 @@
 -- database: presto; groups: qe, horology_functions
-SELECT timezone_hour(TIMESTAMP '2001-08-22 03:04:05.321' at time zone 'Asia/Oral'),
-       timezone_minute(TIMESTAMP '2001-08-22 03:04:05.321' at time zone 'Asia/Oral')
\ No newline at end of file
+SELECT timezone_hour(TIMESTAMP '2001-08-22 03:04:05.321+00:00' at time zone 'Asia/Oral'),
```

Comment:
between time when there was last travis execution of product tests PR execution on travis and time when it was merged there was a commit which changed the semantic of `AT TIME ZONE` (see https://github.com/facebook/presto/commit/834f8c359c5e6bcdefde4ff4a8fb7bebd91ba180). 

In order to prevent such situation in future we would need to run travis just before merge.